includes/admin/class-sensei-onboarding.php

```diff
@@ -97,4 +99,60 @@ public function onboarding_page() {
 		<?php
 	}
 
+	/**
+	 * Check if should show help tab or not.
+	 *
+	 * @param string $screen_id Screen ID to check if should show the help tab.
+	 *
+	 * @return boolean
+	 */
+	private function should_show_help_screen( $screen_id ) {
+		return 'edit-course' === $screen_id;
+	}
+
+	/**
+	 * Add onboarding help tab.
+	 *
+	 * @access private
+	 */
+	public function add_onboarding_help_tab() {
+		$screen                 = get_current_screen();
+		$setup_wizard_completed = get_option( 'sensei_setup_wizard_completed' );
```

Comment:
I added this check to check if the user completed the wizard. If y'all like the idea, I'll create a card to create this option as `true` when the user complete the wizard.

Reply:
To clarify, this Help tab should always be displayed, regardless of whether or not they have completed it.

Reply:
Oh, thank you for the clarification @donnapep! I'll remove this feature.

Reply:
https://github.com/Automattic/sensei/pull/3093/commits/7f8ea8ee8abafdeb870f8934ab898a8f31c79c6e

Reply:
> This should probably be also here when the user dismissed the wizard or the notice suggesting it.

Agreed. Always visible and not conditional on anything, which will make it easier for us to launch it as well.